t/app/controller/report_new.t

```diff
@@ -1992,6 +1992,61 @@ subtest "check map click ajax response for inspector" => sub {
         $extra_details = $mech->get_ok_json( '/report/new/ajax?latitude=55.952055&longitude=-3.189579' );
     };
     like $extra_details->{category}, qr/data-role="inspector/, 'category has correct data-role';
+    like $extra_details->{category}, qr/data-prefill="0"/, 'inspector prefill not set';
```

Comment:
Just in the interest of this already being a long test suite, I think these tests are covered by the JS test and so aren't needed.